ext/opcache/ZendAccelerator.c

```diff
@@ -2957,11 +2957,18 @@ void accel_shutdown(void)
 
 void zend_accel_schedule_restart(zend_accel_restart_reason reason)
 {
+	const char *zend_accel_restart_reason_text[ACCEL_RESTART_USER + 1] = {
+		[ACCEL_RESTART_OOM] = "out of memory",
+		[ACCEL_RESTART_HASH] = "hash overflow",
+		[ACCEL_RESTART_USER] = "user",
+	};
```

Comment:
This is C99 syntax, which we currently don't allow. This should be rewritten without designated initializers.